RNTester/js/examples/Accessibility/AccessibilityExample.js

```diff
@@ -512,17 +512,47 @@ class AccessibilityActionsExample extends React.Component {
     );
   }
 }
-class ScreenReaderStatusExample extends React.Component<{}> {
+class DisplayOptionsStatusExample extends React.Component<{}> {
```

Comment:
Enhancing the RNTester examples to exercise the features is awesome.   But we need to 1) contribute the change back upstream, and 2) mark the deltas from upstream with the `// [TODO` , `// ]TODO` comment blocks.    In this case, as they are changes that work equally well for iOS and should be contributed upstream, the comments can be `// [TODO(OSS Candidate ISS#2710739)` and `// ]TODO(OSS Candidate ISS#2710739)`.   You can do a diff of this file against the version in fb/master.